_docs/shortcuts-lab.md

```diff
@@ -28,3 +28,7 @@ Add an object into a group while preserving the group’s animations. First, sel
 ## <img src="{{ site.baseurl }}/img/docs/shortcuts-lab/image_7.png" width="44"> Fill Slide
 
 Fills any image(s) to the slide while maintaining aspect ratio. Any area that exceeds the slide boundaries are automatically cropped. First, select any image(s), then right-click on your selection, and choose this option in the submenu `PowerPointLabs` to make your selection fill the slide.
+
+## <img src="{{ site.baseurl }}/img/docs/shortcuts-lab/image_8.png" width="44"> Export Slide As Image
+
```

Comment:
It might be better to add a screenshot here to show where is the `slide thumbnail`, although I noticed that the other shortcut lab features don't have images.